scripts/quantile_summarize_geoid_level.py

```diff
@@ -0,0 +1,69 @@
+#!/opt/spark/bin/spark-submit
+"""
+A faster and distributed version of QuantileSummarizeGeoidLevel.R, which uses Apache Spark as its execution engine
+
+This script can be executed from the command line in the container by running Apache Spark locally:
+% /opt/spark/bin/spark-submit --driver-memory 20g --executor-memory 20g \
+        quantile_summarize_geoid_level.py USA_None -o usa_none.csv
+"""
+
+import itertools
+import pathlib
+
+import click
+import numpy as np
+from pyspark.sql import functions as F, SparkSession, SQLContext, Window
+
+PROBS = np.concatenate([[0.01, 0.025], np.arange(start=0.05, stop=0.95, step=0.05), [0.975, 0.99]])
+METRICS = ["hosp_curr", "cum_death", "death", "infections", "cum_infections", "hosp"]
+
+spark = (SparkSession.builder.appName("quantile report").getOrCreate())
+sc = spark.sparkContext
+sc.setLogLevel("WARN")
+sqlContext = SQLContext(sc)
+
+
+@click.command()
+@click.option("-o", "--output", type=pathlib.Path, required=True,
+              help="write output to this directory")
+@click.option("--start_date", type=click.DateTime(formats=["%Y-%m-%d"]),
+              default="2020-01-01", show_default=True,
+              help="earliest date to include")
+@click.option("--end_date", type=click.DateTime(formats=["%Y-%m-%d"]),
+              default="2022-01-01", show_default=True,
+              help="latest date to include")
+@click.argument("input_paths", type=click.Path(), nargs=-1, required=True)
+def process(input_paths, output, start_date, end_date):
+    paths = itertools.chain(*(pathlib.Path("hospitalization/model_output").glob(p + "/**/*.parquet")
```

Comment:
Can you add a name_filter argument? (See original script) It's usually "low", "med", or "high" for death rates. So that just the low_*.parquet files are processed.

And throw a descriptive error if an input_path is bad?

Reply:
done

Reply:
done